src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
 spring.data.mongodb.uri=${MONGODB_URL}/transaction_confirmation_statement_submissions
+confirmation.statement.no.updates=**Confirmation statement** made on {made up date} with no updates
```

Comment:
I thought this would be in the environment properties (chs-config) , maybe @jsimons1989 can confirm